docs/training-guide.md

```diff
@@ -146,11 +146,19 @@ for example [`teacher.train.yml`] and in the [`train.py`] script.
 
 ### Model training
 
+#### Student architecture
+
+"base" or "tiny" based on [Bergamot configurations](https://github.com/browsermt/students/tree/master/train-student/models).
```

Comment:
Suggestion: It would be nice to provide additional context: https://aclanthology.org/D19-5632.pdf

And note that the names differ from the paper due to different decoder depths.

Reply:
I'm actually not sure from which paper those configs were taken as they don't match the ones in this paper. So I'm referring to the student repo everywhere.